pkg/detectors/mongodb/mongodb.go

```diff
@@ -85,6 +87,30 @@ func isErrDeterminate(err error) bool {
 }
 
 func verifyUri(uri string, timeout time.Duration) error {
+	parsed, err := url.Parse(uri)
+	if err != nil {
+		return err
+	}
+
+	params := url.Values{}
+	for k, v := range parsed.Query() {
```

Comment:
The current MongoDB pattern needs to be updated to handle query parameters  (see #1550).

Reply:
Oh, I'll take a look. Thank you.